dependencies/ACE/CMakeLists.txt

```diff
@@ -82,6 +82,7 @@ if (${WIN32})
   set ( _CONFIGURE_COMMAND ${_COMMAND_PREFIX}
                              ${CMAKE_COMMAND} -E env ACE_ROOT=<SOURCE_DIR> SSL_ROOT=${OPENSSL_ROOT}
 							 ${MPC} ${MPC_FLAGS} -name_modifier "*_${_TYPE}_static" -static
+                             -value_template WindowsTargetPlatformVersion=${CMAKE_SYSTEM_VERSION}
```

Comment:
Does not match formatting of the surrounding lines.

Reply:
Fixed.